src/assets/scss/components/_imageinput.scss

```diff
@@ -0,0 +1,32 @@
+.image-input {
+    input[type=radio]:checked + .image-input__image {
+        border-radius: 10%;
+        box-shadow: inset 0 0 0 5px $accent; 
+        padding: 10px;
+        transition: all .5s ease-in-out;
```

Comment:
Unless this `transition` is different for when the box is checked or not, you can safely include it within `&__image` once and not have to redefine it for each interaction state:

```
&__image {
  transition: all .5s ease-in-out;
```

And then remove it from lines `6` and `28`.